packages/hello-world-react/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "hello-world-react",
+  "version": "1.4.0-dev",
+  "mobify": {
```

Comment:
To be safe, I think it'd be nice to set `"private": true,` in the package.json, to avoid accidentally publishing the template as a package.

Reply:
Very good suggestion! Thanks!